pkg/security/ebpf/c/activity_dump.h

```diff
@@ -195,16 +203,22 @@ __attribute__((always_inline)) void cleanup_traced_state(u32 pid) {
 #define NO_ACTIVITY_DUMP       0
 #define IGNORE_DISCARDER_CHECK 1
 
+union cgroup_comm_combo {
+    char cgroup[CONTAINER_ID_LEN];
```

Comment:
what about container_id ?

Reply:
Not sure I understand what you mean, the goal here is to reduce the stack by combining both char arrays since they are not used at the same time

Reply:
Oh you mean changing the name to `container_id` ?

Reply:
I can change this yes

Reply:
should be good now